packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -1333,24 +1333,27 @@ Adds an OAuth 2.0 client application. This application is only available to the
+| response_types                                                                                            | Array of OAuth 2.0 response type strings                                                                                                                                                                                   | Array of `code`, `token`, `id_token`                                                           | TRUE       | FALSE    | TRUE       |
+| tos_uri                                                                                                   | URL string of a web page providing the client's terms of service document                                                                                                                                                  | URL                                                                                            | TRUE       | FALSE    | FALSE      |
+| refresh_token <ApiLifecycle access="ea" />                                                                | Refresh token configuration                                                                                                                                                                                                | [Refresh Token object](#refresh-token-object)                                                                                            | TRUE       | FALSE    | TRUE      |
+| jwks_uri <ApiLifecycle access="ea" />                                                                     | URL string that references a [JSON Web Key Set](https://tools.ietf.org/html/rfc7517#section-5) for validating JWTs presented to Okta.                                                                                      | String                                                                                        | TRUE       | FALSE    | TRUE      |
+| backchannel_token_delivery_mode <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />                 | The delivery mode for Client-Initiated Backchannel Authentication. Valid types include `poll`, `ping` and `push`, but only `poll` is currently supported.                                                                  | String                                                                                        | TRUE       | FALSE    | TRUE      |
+| backchannel_authentication_request_signing_alg <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />  | The signing algorithm for Client-Initiated Backchannel Authentication signed requests. Signed requests will not be supported if this value is not set.                                                                     | String                                                                                        | TRUE       | FALSE    | TRUE      |
```

Comment:
When you say "aren't supported" does that mean that an error is sent and the request fails? Or does it just ignore that value?
```suggestion
| backchannel_authentication_request_signing_alg <ApiLifecycle access="ie" /> <ApiLifecycle access="ea" />  | The signing algorithm for Client-Initiated Backchannel Authentication signed requests. If this value isn't set, signed requests aren't supported.                                                                     | String                                                                                        | TRUE       | FALSE    | TRUE      |
```

Reply:
If backchannel_authentication_request_signing_alg is not set and we sent a signed request, the request fails.

Reply:
ok - suggest the updated edit based on your response.

Reply:
@susanharper-okta The request will fail only in case we send signed request and backchannel_authentication_request_signing_alg is not set.
If we dont have backchannel_authentication_request_signing_alg set and we send an unsigned request, it would work fine.

Reply:
updated